<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>foryouos</class>
 <widget class="QMainWindow" name="foryouos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>468</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>foryouos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Main_Widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="sideBar" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color: rgb(228, 228, 228);
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QToolButton" name="ToDoactionBtn">
            <property name="minimumSize">
             <size>
              <width>76</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 70px;
    min-height: 70px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);
	background-color: rgb(246, 246, 246);	
}
</string>
            </property>
            <property name="text">
             <string>ToList</string>
            </property>
            <property name="icon">
             <iconset resource="photo.qrc">
              <normaloff>:/photo/-to-do.png</normaloff>:/photo/-to-do.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QToolButton" name="ChineseColorBtn">
            <property name="minimumSize">
             <size>
              <width>76</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 70px;
    min-height: 70px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

</string>
            </property>
            <property name="text">
             <string>传统色</string>
            </property>
            <property name="icon">
             <iconset resource="photo.qrc">
              <normaloff>:/photo/color.png</normaloff>:/photo/color.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="MainactionBtn">
            <property name="styleSheet">
             <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 70px;
    min-height: 70px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

</string>
            </property>
            <property name="text">
             <string>首页</string>
            </property>
            <property name="icon">
             <iconset resource="photo.qrc">
              <normaloff>:/photo/Tools.png</normaloff>:/photo/Tools.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Right_Main_Widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="Hide_Btn_Widget" native="true">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-top-right-radius:0px;
border-bottom-left-radius:10px;
border-bottom-right-radius:0px;
border: 0px solid #ccccd6;</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>0</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <spacer name="Vertial_Hide_Btn">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>700</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="HideSideBtn">
                 <property name="minimumSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: transparent;
border: 0px solid #388E3C;
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="photo.qrc">
                   <normaloff>:/photo/circle.png</normaloff>:/photo/circle.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="MainStackWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-top-left-radius:0px;
border-top-right-radius:10px;
border-bottom-left-radius:0px;
 border-bottom-right-radius:10px;
border: 0px solid #ccccd6;</string>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="Main_Page">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QWidget" name="Main_Widget_Page" native="true">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:10px;
border: 1px solid #ccccd6;</string>
                </property>
                <widget class="QLabel" name="label">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>20</y>
                   <width>401</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Harrington</family>
                   <pointsize>36</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Botton</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
                <widget class="QGroupBox" name="groupBox">
                 <property name="geometry">
                  <rect>
                   <x>120</x>
                   <y>120</y>
                   <width>301</width>
                   <height>361</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(85, 170, 127);</string>
                 </property>
                 <property name="title">
                  <string>未完成任务</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="ChineseColorPage">
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::Shape::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Shadow::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="TODOPage">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="ToDoWidget" native="true">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
 border-top-left-radius:0px;
 border-top-right-radius:10px;
 border-bottom-left-radius:0px;
 border-bottom-right-radius:10px;
border: 1px solid #ccccd6;</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0" colspan="3">
                  <widget class="QWidget" name="tolisttitleWidget" native="true">
                   <property name="styleSheet">
                    <string notr="true">border: 0px solid #ccccd6;</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="pushButton_3">
                      <property name="minimumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">image: url(:/photo/Tools.png);</string>
                      </property>
                      <property name="text">
                       <string>首</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="font">
                       <font>
                        <family>Harrington</family>
                        <pointsize>20</pointsize>
                        <italic>true</italic>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">color:rgb(80, 121, 255);
border: 0px solid #ccccd6;
</string>
                      </property>
                      <property name="text">
                       <string>Better to do than to say!</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="minimumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">border:1px solid #ccccd6;
image: url(:/photo/permission.png);</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QFrame" name="frame_2">
                   <property name="styleSheet">
                    <string notr="true">border: 0px solid #ccccd6;</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <property name="spacing">
                     <number>6</number>
                    </property>
                    <item>
                     <widget class="QListView" name="ToDoList">
                      <property name="styleSheet">
                       <string notr="true">border: 0px solid #ccccd6;</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_2" native="true">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">border: 1px solid #ccccd6;
background-color: rgb(0, 170, 127);
 border-top-left-radius:10px;
 border-top-right-radius:10px;
 border-bottom-left-radius:10px;
 border-bottom-right-radius:10px;</string>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QPushButton" name="pushButton">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>50</width>
                           <height>50</height>
                          </size>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">border-top-right-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:10px;
border-bottom-left-radius:10px</string>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="icon">
                          <iconset theme="QIcon::ThemeIcon::DocumentOpenRecent"/>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>30</height>
                          </size>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">border: 0px solid #ccccd6;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Mainaction"/>
   <addaction name="ToDoaction"/>
   <addaction name="ChineseColor"/>
  </widget>
  <action name="ChineseColor">
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/photo/color.png</normaloff>:/photo/color.png</iconset>
   </property>
   <property name="text">
    <string>中国传统色</string>
   </property>
  </action>
  <action name="Mainaction">
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/photo/Tools.png</normaloff>:/photo/Tools.png</iconset>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
   <property name="toolTip">
    <string>进入首页</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="ToDoaction">
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/photo/-to-do.png</normaloff>:/photo/-to-do.png</iconset>
   </property>
   <property name="text">
    <string>代办事项</string>
   </property>
   <property name="toolTip">
    <string>代办事项</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
